src/seedu/addressbook/data/person/Address.java

```diff
@@ -55,4 +55,9 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+    
+    @Override
+    public String getPrintableString() {
+        return " Address: " + value;    
```

Comment:
Maybe the preceding whitespace before "Address" should be omitted. If only the Printable String of Address is required, we should be able to have it ready without the need to trim it.